samples/sample_multi_transcode/src/pipeline_transcode.cpp

```diff
@@ -550,7 +550,8 @@ mfxStatus CTranscodingPipeline::DecodeOneFrame(ExtendedSurface *pExtSurface)
         {
             pExtSurface->pSurface = GetFreeSurface(false, MSDK_SURFACE_WAIT_INTERVAL);
             sts = m_pBSProcessor->GetInputFrame(pExtSurface->pSurface);
-            MFX_CHECK_STS(sts);
+            if(sts != MFX_ERR_NONE)
```

Comment:
@dylanintel , could you do it via "MSDK_BREAK_ON_ERROR(sts);" ?

Reply:
hi @dmitryermilov , The issue is that sample_multi_transcode don't  support from i420/nv12 to hevc, m_mfxDecParams.mfx.FrameInfo.PicStruct = MFX_PICSTRUCT_PROGRESSIVE; The code aims to solve this issue. In addition, there is another issue, it will print the MFX_ERR_MORE_DATA error, But the task could be finished successfully. The root cause is that it will return  MFX_ERR_MORE_DATA when reading the end of yuv file. I think it is the normal phenomenon. Because it will deal with the error and set the bEndOfFile as true in CTranscodingPipeline::Transcode function. The function MFX_CHECK_STS will print the error if the status !=  MFX_ERR_NONE. So I  return an error instead of printing and returning an error.